### ISSUE TITLE
Msg for unavailable video shows book name  as undefined for OT books 

### ISSUE THREAD

**Issue by AnnCherianA** [Bridgeconn/VachanOnline-v2#427]
On selecting  videos  for OT books  in a language  which has only NT books available ,the book name is shown as undefined.

Steps 
Open ESV English and choose Genesis 1 from book drop down
Select videos on the right panel 
From the drop down select any language that has only NT books available on the website etc Haryanvi / Dogri / Nagamese etc

The message No videos available in Haryanvi/ Dogri/ Nagamese for undefined is shown. 

The book name is shown as undefined for unavailable OT books

Expected: It display the OT book or display a message such as videos are available only for NT books .

**Comment by AnnCherianA** [comment]
@VineethaVi please add the screenshot shared. Thank you

**Comment by VineethaVi** [comment]
![MicrosoftTeams-image (71)](https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/fb62a970-ae6f-42cc-8e6e-d45a15a38cf9)

**Comment by RevantCI** [comment]
- [ ] Fix issue ( 4 hours)
- [ ] Code Review - Beena ( 2 hours)
- [ ] QA ( 6 hours)

**Comment by AnnCherianA** [comment]
Updating the QA hours (reducing to 3 hours -1.5 hours on website and mobile each  would be sufficient)

**Comment by AnnCherianA** [comment]
**Test result** 
Website 
The issue has been fixed . It gives the book name. However, the format for the message is different for each language. It would be better if an uniform pattern is followed.
please find the messages attached. 

Dogri -  For  Dogri videos , in the message  the OT Book name ie Genesis is given in English 
![image](https://github.com/Bridgeconn/VachanOnline-v2/assets/74647279/511384a0-3f2d-4a9e-886e-abd2bfff3251)
Similarly for   Haryanvi videos, in the message  the OT book name ie Genesis is given in English 
![image](https://github.com/Bridgeconn/VachanOnline-v2/assets/74647279/26b2ee81-62d3-4458-ba71-02ed6dad025d)
For  Assamese vidoes, for the unavailable message , the book name Genesis is given in Assamese itself which is good  ( as shown in the screenshot ) but it would be better if this pattern is followed throughout. 

![image](https://github.com/Bridgeconn/VachanOnline-v2/assets/74647279/aa8d6588-1937-4834-b7b5-48c8ba0efc1c)
Again ,noted the same pattern for Marathi where the book name Psalms is shown in Marathi itself.  ( स्तोत्रसंहिता)
![image](https://github.com/Bridgeconn/VachanOnline-v2/assets/74647279/08934c72-1e46-4d89-ad62-8631ec279542)

- Please find the full list of languages where the book name is shown in the selected language itself  in the video unavailable message  :  (preferred pattern)

**Assamese, Marathi, Nepali, Odia, Punjabi ,Urdu**

- The list of languages for which the book name is given in English in the video unavailable message 

( Please take  look  @RevantCI and check if for the below languages, the  book name can be shown in the selected /original language itself
**-Haryanvi, Dogri, Nagamese**

**Comment by AnnCherianA** [comment]
Mobile browser 👍 
The message no videos are available flashes for a few seconds before the user reads the message and displays videos of Matthew 1
Please increase the time for which the message is shown. 
Screenshot (will attach soon)

**Comment by AnnCherianA** [comment]
Screenshot for Mobile browser
![image](https://github.com/Bridgeconn/VachanOnline-v2/assets/74647279/c44076cb-8079-495a-ace5-64eb81de7f58)

**Comment by AnnCherianA** [comment]
Moving to in  progress for feedback for the suggestions requested from the dev team

**Comment by RevantCI** [comment]
Regarding the point raised by @AnnCherianA  above for the Bible book name in the message
In the video not available message the format is `No videos available in {language} for {book}`
Currently we do not have all the bible books as the complete bibles have not been translated for all minority languages.
Those that have been we have extracted the bible book names and put in the vachan engine.
We have asked the Translation team for the missing bible book names but have not been provided yet.
The work around is that we have added the English bible book names in the code itself and if for a language the book name is not available we show the English bible book name as a fallback.
